package advanced;

public class TryCatchRuntime {
	public static void main(String[] args){
		try{
			throw new ArithmeticException("Hello");
		}
		finally{
		}
	}
}
